templates/partners/kubernetes.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
+    <div class="p-strip--suru-background">
+      <div class="row">
+        <div class="col-6">
+          <h1>Charmed Kubernetes partners</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore deleniti eaque repellendus, impedit nobis veniam, pariatur qui quae laboriosam dolores doloremque iusto nesciunt nihil doloribus officia iste modi, repellat consectetur!</p>
+        </div>
+        <div class="col-6 u-align--center u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/0038a560-charmed-kubernetes-partners-cloud.svg" alt="charmed kubernetes image">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip is-deep u-extra-space">
+    <div class="row u-sv3">
+      <p class="p-heading--two">Technology and product partners</p>
+    </div>
+    <div class="row">
+      <div class="p-card--no-border col-4">
+        <img class="p-card__thumbnail-large" src="https://assets.ubuntu.com/v1/8f878ba0-rancher-logo-stacked-color.svg" alt="rancher image">
+        <h3 class="p-card__title" style="padding-top: 3rem;">Rancher Labs</h3>
+        <p class="p-card__content u-sv3">Rancher provides a turn-key application delivery platform, built on Ubuntu and the Charmed Distribution of Kubernetes (CDK). Maximise developer velocity, integrate CI/CD, and ease the path from development into production with enterprise-calibre container management. Canonical delivers a cloud-native platform in partnership with Rancher Labs.</p>
+        <a href="" class="p-button--positive">Watch the Introduction webinar</a>
+      </div>
+      <div class="p-card--no-border col-4">
+        <!-- Update image -->
+        <img class="p-card__thumbnail-large" src="https://assets.ubuntu.com/v1/8f878ba0-rancher-logo-stacked-color.svg" alt=" image">
```

Comment:
Is this a placeholder image? It doesn't match the design (I know there's some lorem ipsum, so wasn't sure). Also, the images are left aligned, but center aligned in the design.